core/src/main/scala/tech/beshu/ror/accesscontrol/factory/JsonConfigStaticVariableResolver.scala

```diff
@@ -20,14 +20,16 @@ import cats.data.NonEmptyList
 import cats.implicits._
 import eu.timepit.refined.types.string.NonEmptyString
 import io.circe.Json
+import tech.beshu.ror.accesscontrol.blocks.variables.VariableCreationConfig
 import tech.beshu.ror.accesscontrol.blocks.variables.startup.StartupResolvableVariableCreator.{createMultiVariableFrom, createSingleVariableFrom}
 import tech.beshu.ror.accesscontrol.show.logs._
 import tech.beshu.ror.providers.EnvVarsProvider
 
 object JsonConfigStaticVariableResolver {
 
   def resolve(json: Json)
-             (implicit envProvider: EnvVarsProvider): Either[NonEmptyList[ResolvingError], Json] = {
+             (implicit envProvider: EnvVarsProvider,
+              config: VariableCreationConfig): Either[NonEmptyList[ResolvingError], Json] = {
```

Comment:
should we move both implicits to the constructor? (like it was mentioned in the case of variable creators?)